static/app/components/assistant/guideAnchor.tsx

```diff
@@ -11,15 +11,15 @@ import {
   registerAnchor,
   unregisterAnchor,
 } from 'sentry/actionCreators/guides';
-import {Guide} from 'sentry/components/assistant/types';
 import Button from 'sentry/components/button';
 import {Body as HovercardBody, Hovercard} from 'sentry/components/hovercard';
 import {t, tct} from 'sentry/locale';
-import GuideStore, {GuideStoreState} from 'sentry/stores/guideStore';
+import GuideStore from 'sentry/stores/guideStore';
+import {useLegacyStore} from 'sentry/stores/useLegacyStore';
 import space from 'sentry/styles/space';
 import theme from 'sentry/utils/theme';
 
-type Props = {
+type Props = React.PropsWithChildren<{
```

Comment:
I dont think this does what you think it does - 
`type PropsWithChildren<P> = P & { children?: ReactNode | undefined };` has optional children type, but I think for this component you want to require children to always be present.